appcoins_net/src/main/java/com/appcoins/net/AppCoinsConnection.java

```diff
@@ -0,0 +1,8 @@
+package com.appcoins.net;
+
+interface AppCoinsConnection {
```

Comment:
If this class has such relation with the ads model, does it make sense to in a different module? Its purpose doesn't exist out of the ads context.